src/lexoRank/lexoRank.ts

```diff
@@ -311,6 +311,69 @@ export class LexoRank {
     return new LexoRank(this.bucket, LexoRank.between(this.decimal, other.decimal));
   }
 
+  public betweenLexoRanks(other: LexoRank, ranksToGenerate: number): LexoRank[] {
+
+    if (!this.bucket.equals(other.bucket)) {
+      throw new Error('Between works only within the same bucket');
+    }
+
+    if (ranksToGenerate == 0) {
+      return [];
```

Comment:
actually, let's throw an exception in this case

Reply:
Thrown exception for this condition.